doc/source/extras/plot_directive.rst

```diff
@@ -34,6 +35,25 @@ Which will be rendered as:
    >>> sphere = pyvista.Sphere()
    >>> out = sphere.plot()
 
+.. note::
+
+   You need to install the following packages to build the interactive scene.
+
+   * `jupyter_sphinx>=0.5.3`
+   * `jupyterlab>=4.0.10`
+   * `osmnx>=1.8.1`
```

Comment:
why `osmnx` ? It is not seem directly relevant to the interactive infrastructure. 
Same with `jupyterlab` I am not sure it is strictly required. Although it does not hurt.

Do we really want to have explicit packages here ? I am sure we will forget to update them at some point. Why not point to the `requirements_docs.txt` file ? OR maybe we can add some comments in that file saying which packages are required in general and which for pyvista specifically.

Reply:
Sorry, I was wrong about needing osmx. We can confirm required package by looking at the following repository I will add a note to the CONTRIBUTION so that when the dynamic plots are updated, this repository will also be updated.
https://github.com/pyvista/pyvista-docs-dynamic

Reply:
I agree with this comment, it will be hard to maintain here.  I like the idea of pointing to the requirements.  Can we instead link to the https://github.com/pyvista/pyvista-docs-dynamic/blob/main/requirements.txt by somehow embedding it into the docs there and providing logical commenting inside?

Reply:
Thanks for your comments. Resolved in 6dfd959a981cc7403d817fa0b8f2313a0e38bada . Please check to see if we have reflected your intentions.